src/features/vsProvider.ts

```diff
@@ -212,11 +221,14 @@ export default class ValeServerProvider implements vscode.CodeActionProvider {
     );
     subscriptions.push(this);
 
-    this.readabilityStatus = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 100);
+    this.readabilityStatus =
+      vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left, 100);
     this.diagnosticCollection = vscode.languages.createDiagnosticCollection();
 
     this.useCLI = configuration.get('vale.core.useCLI', false);
-    this.stylesPath = await utils.getStylesPath(this.useCLI);
+    // this.stylesPath = await utils.getStylesPath({
+    //   usingCLI: this.useCLI,
+    // });
```

Comment:
During activation, I don't see how we can get the information about the workspace folder, which is required to get the correct binary + config paths.

Is setting the `stylesPath` necessary at this point, given that it's also set on `runVale`?